samples/msal-angular-v2-samples/angular10-browser-sample/src/app/app.module.ts

```diff
@@ -24,6 +27,16 @@ function MSALInstanceFactory(): IPublicClientApplication {
   });
 }
 
+function MSALInterceptorConfigFactory(): MsalInterceptorConfig {
+  const protectedResourceMap = new Map<string, Array<string>>()
+  protectedResourceMap.set('https://graph.microsoft.com/v1.0/me', ['user.read'])
```

Comment:
This should probably be done in a config file right?

Reply:
`app.module.ts` is essentially a config file for the app. We could put it in a separate file and have this module pull it in, but easier to just put it here.